Resources/Prototypes/Entities/Structures/Decoration/bonfire.yml

```
@@ -31,6 +31,28 @@
     sound:
       path: /Audio/Ambience/Objects/fireplace.ogg
   - type: AlwaysHot
+  - type: TriggerOnCollide
+    fixtureID: ignition
+  - type: Fixtures
```

```suggestion
  # Harmony. Bonfire improvements below
  - type: TriggerOnCollide
    fixtureID: ignition
  - type: Fixtures
```

Upstream file change lacks comment. Add Harmony comment that explains the change before the `TriggerOnCollide` component